### ISSUE TITLE
How to fine-tuning pre-trained model with my own data?

### ISSUE THREAD

**Issue by aquachieh** [takiyu/hyperface#11]
Hi,
I want to improve the gender accuracy,
so I'd like to finetune your model (model_epoch_190) with my dataset 
(some croped face with 21 landmarks and gender label)

but I still don't know how to start it,
should I add my dataset to aflw.sqlite ?
can you give me some tips ?

Any help will be highly appreciated,Thanks!

**Comment by takiyu** [comment]
Thank you for using my implementation.
You can setup Chainer model and deserialize from `model_epoch_190` with the following code.
```
model = models.HyperFaceModel()
chainer.serializers.load_npz('path of model_epock_190', model)
```

`aflw.sqlite` is a cache file for AFLW dataset.
When you use another dataset, it is not needed, and you have to write another dataset wrapper (Chaniner's dataset class).

To train with Chainer's model, please read tutorials of Chainer.
http://docs.chainer.org/en/stable/tutorial/index.html

**Comment by aquachieh** [comment]
I will try you suggestions soon ,Thank you very much!
